lib/src/gutter.dart

```diff
@@ -9,6 +9,12 @@ class Gutter extends StatelessWidget {
     Key? key,
   ]) : super(key: key);
 
+  static List<Widget> separateChildren({
+    required List<Widget> children,
+    required double extent,
+  }) =>
+      children.separate(extent);
```

Comment:
The purpose of Gutter is that extent is optional. I would consider it like

``` 
static List<Widget> separateChildren({
    required List<Widget> children,
    double? extent,
 }) =>
      children.separate(extent);

And use `Gutter` instead of `Gap`

Reply:
That makes sense to me. Already updated!